docs/preprocess-templates/expansion_logpoltens.ipynb

```diff
@@ -0,0 +1,940 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        ii_LS_alm[ii] = hp.almxfl(alm_log_pol_tens_fullsky[ii] , (1.-sig_func)**.2 ) 
</pre><p>@giuspugl it seems we are applying the sigmoid function to the small scales twice, first when we create <code>small_scales_input_cl</code>  and then with <code>almxfl</code>.  What is the reason for that?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/galsci/pysm/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
Good catch @zonca ! that's a bug !  to fix it I would simply do : 

```
np.random.seed(777)
#filter small scales 
small_scales_input_cl = [
    1 \
    * model(ell, A_fit[pol], gamma_fit[pol]) \
    * sigmoid(ell, ell_fit_high[pol], ell_fit_high[pol]/10)  \
    / cl_norm \
    for pol in spectra_components
]
ii_SS_alm =hp.synalm(small_scales_input_cl+ [np. zeros_like(small_scales_input_cl[0])]*3 ,lmax=lmax, new=True )
#filter large scales 
alm_log_pol_tens_fullsky = hp.map2alm(log_pol_tens_varres   , lmax=lmax, use_pixel_weights=True)
ii_LS_alm=np.empty_like(alm_log_pol_tens_fullsky)

for ii, pol in enumerate(spectra_components ):
    
    sig_func =  sigmoid(ell, x0= ell_fit_high[pol], width= ell_fit_high[pol]/10)
    ii_LS_alm[ii] = hp.almxfl(alm_log_pol_tens_fullsky[ii] , (1.-sig_func)**.2 ) 

log_ss =hp.alm2map( ii_SS_alm , nside=nside ) 
log_ss[0] *=modulate_amp 
log_ss[1:] *=modulate_amp_pol

log_ls =hp.alm2map(ii_LS_alm, nside=nside ) 
ii_map_out2 = log_ss + log_ls 


```

Reply:
I checked the maps and the relative difference is at order of 0.1%  ... We definitely want to fix this..

Reply:
ok, fixed in f5da8cb